frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/row/action.dart

```diff
@@ -16,16 +16,22 @@ import '../../layout/sizes.dart';
 class RowActions extends StatelessWidget {
   final String viewId;
   final RowId rowId;
+  final String? groupId;
   const RowActions({
     required this.viewId,
     required this.rowId,
+    this.groupId,
     Key? key,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     return BlocProvider(
-      create: (context) => RowActionSheetBloc(viewId: viewId, rowId: rowId),
+      create: (context) => RowActionSheetBloc(
+        viewId: viewId,
+        rowId: rowId,
+        groupId: groupId,
+      ),
       child: BlocBuilder<RowActionSheetBloc, RowActionSheetState>(
```

Comment:
I would recommend moving the BlocProvider to the parent widget. When both the BlocProvider and the BlocBuilder are in the same widget, it makes it harder to test since you cannot mock the BlocProvider's values.

Reply:
Adding this will make the unit tests that you need to land this easier 👍